.travis.yml

```diff
@@ -408,7 +408,7 @@ cache:
     - $CACHE_DIR
 
 before_install:
-  - if [ "$(uname)" = "Darwin" ] ; then export NUM_PROCESSORS=$(sysctl -n hw.physicalcpu); else export NUM_PROCESSORS=$(nproc); fi
+  - if [ "$(uname)" = "Darwin" ] ; then export NUM_PROCESSORS=1; else export NUM_PROCESSORS=$(nproc); fi
```

Comment:
What's the motivation for this change? (apologies if this has been discussed, just refer me to a previous discussion if so). At any rate, a comment explaining the change would be good.

Reply:
This change was made due to problems of launching unit tests in parallel on Mac platform, see the thread https://ripplelabs.slack.com/archives/C02GE9NBA/p1586457327294700

Done comment on .travis.yml.